bots/lambda_detach_external_layer.py

```diff
@@ -0,0 +1,73 @@
+'''
+## lambda_detach_external_layer
+Log.ic bot only
+What it does: Detaches a layer version from a lambda function if it was added from an external account.
+Usage: AUTO: lambda_detach_external_layer
+Limitations: The bot will stop running if the proper 'UpdateFunctionConfiguration20150331v2' event is not found.
+'''
+
+import boto3
+import bots_utils
+import botocore.exceptions
+import json
+
+LAYER_NAME_PLACEMENT = 6
+EVENT_NAME = 'UpdateFunctionConfiguration20150331v2'
```

Comment:
Is it the only event relevant for updating the lambda?

Reply:
Lior told me it's ok to use 'UpdateFunctionConfiguration20150331v2' only